cmd/nodeup/main.go

```diff
@@ -41,13 +41,15 @@ func main() {
 	}
 	fmt.Printf("nodeup version %s%s\n", kops.Version, gitVersion)
 
-	var flagConf string
+	var (
+		flagConf     string
+		flagCacheDir string
+		flagRootFS   string
+		flagRetries  int
+	)
 	flag.StringVar(&flagConf, "conf", "node.yaml", "configuration location")
-	var flagCacheDir string
```

Comment:
This code is actually an older style altogether.

Something I have found that works well with flags is this pattern:

`
flag1 := "default-flag1"
flag.StringVar(&flag1, "one", flag1, "help one")
flag2 := -1
flag.IntVar(&flag2, "two", flag2, "help two")
```

In other words, we put the flag default values into the variable itself.  This is much nicer because it lets us move to an options structure more readily, which we tend to end up doing in the long run.

Do you want to refactor it in this direction instead?  (Or even go to an Options struct, which would also group the variables)